packages/injective-protobuf/README.md

```diff
@@ -5,17 +5,29 @@ Rust protobuf files generation for Injective messages.
 ## Prerequisites
 
 - `git submodule update --init`. Downloads the `sdk-go` repository, to reference the protobuf files under `./proto`.
-- The files under `./third_party` need to be manually kept in sync with the ones from `injective-core` at the moment.
+- buf: `brew install bufbuild/buf/buf` on macOS or even Linux. See https://buf.build/docs/installation/ for detailed installation
+    instructions.
 
 ## Usage
 
-`cargo build`
+- `./scripts/protoexport.sh`
 
-Generates the Rust protobuf files under `./src/proto`.
+  Downloads / Updates Cosmos SDK protobuf files under `./third_party/proto/`, using `buf`.
+
+- `cargo build`
+
+  Generates the Rust protobuf files under `./src/proto`.
 
 ## Maintenance
 
-- `git submodule update --remote` needs to be run in order to update the local reference to `sdk-go`.
+- The specific version / tag of the Cosmos SDK proto files should be indicated in the `protoexport.sh` file.
```

Comment:
where/how exactly?

Reply:
In protoexport.sh, the line

`buf export https://github.com/cosmos/cosmos-sdk.git --output ./third_party/proto`

can probably point to a url for a specific tag of the cosmos-sdk repo.

I'm not exactly sure how are you managing these resources, so, I'm not in the best position to define this.

Reply:
I mean: this depends on Injective release process, that is (mostly) unknown to me at the moment.

Reply:
@maurolacy We maintain our SDK fork here: https://github.com/InjectiveLabs/cosmos-sdk. So I think it's fine just letting the user decide which tag, but can you add a concrete example into how to do this in the README?

I am approving and merging already, but you can do this in a follow up.